client/services/code_evaluators/PythonCodeRunnerService.js

```diff
@@ -34,7 +34,13 @@ tie.factory('PythonCodeRunnerService', [
       runCodeAsync: function(code) {
         clearOutput();
         Sk.configure({
-          output: addOutputLine
+         output: addOutputLine,
+         read: function(name) {
```

Comment:
Maybe add brief comment explaining that this snippet enables importing Python stdlib modules, such as `time`.